pom.xml

```diff
@@ -296,12 +296,12 @@
     <skipRunTests>false</skipRunTests>
 
     <!-- When bumping the JDK version run both:
-      ./mvnw -P jmods -Dos.arch=x86_64
-      ./mvnw -P jmods -Dos.arch=aarch64
+      ./mvnw -P jmods -Dos.arch=x86_64 package -DskipTests=true
```

Comment:
maybe we should test also for macos arm? since it's used by developers, and sometimes I have seen a small delay until it's available